src/main/scala/org/camunda/feel/impl/builtin/NumericBuiltinFunctions.scala

```diff
@@ -12,7 +12,7 @@ object NumericBuiltinFunctions {
     "decimal" -> List(decimalFunction, decimalFunction3),
     "floor" -> List(floorFunction),
     "ceiling" -> List(ceilingFunction),
-    "abs" -> List(absFunction),
+    "abs" -> List(absFunctionNumber, absFunctionN),
```

Comment:
We can declare the functions directly here.

```suggestion
    "abs" -> List(absFunction(paramName = "number"), absFunction(paramName = "n")),
```